targets/x86-legacy

```diff
@@ -0,0 +1,10 @@
+include 'x86.inc'
+
+packages {
+        '-kmod-gpio-nct5104d',
+        '-kmod-leds-gpio',
+        '-kmod-leds-apu2',
+}
```

Comment:
Should these package removals also be included upstream? Or are they not relevant anymore because of the move to the new image build code in OpenWrt master?

Reply:
Without these excludes I get:
```
Configuration failed:
 * unable to enable package 'kmod-gpio-nct5104d'
 * unable to enable package 'kmod-leds-gpio'
 * unable to enable package 'kmod-leds-apu2'
make: *** [Makefile:177: config] Fehler 1
```
As these packages are available for other subtargets it would be bad to exclude them for all other x86-subtargets.

Reply:
Ah, these are Gluon additions from `x86.inc`. I think we should just not include `x86.inc`, as we already do for `x86-geode`.

Reply:
@NeoRaider have you intentionally marked your remark as resolved?

Reply:
Yes, this was discussed on IRC, and I'm okay with the current solution.

Reply:
@rotanid IMHO opinion we do not need to look carefully after the image size for the x86 targets. Therefore I see no problem to have much more packages than actually useful on x86 targets. On the other hand I see a problem to have multiple different package lists like currently with x86-geode not using x86.inc. If there is a need for some additional package it currently needs to be added to x86.inc and x86-geode.